pallets/migration/src/lib.rs

```diff
@@ -76,28 +85,28 @@ pub mod pallet {
 
 		/// WeightInfo
 		type WeightInfo: WeightInfo;
+
+		/// The call filter that will be used when pallet is in an ongoing migration
+		type OngoingFilter: Contains<<Self as frame_system::Config>::Call>;
+
+		/// The call filter that will be used when pallet has finalize the migration
+		type FinalizedFilter: Contains<<Self as frame_system::Config>::Call>;
+
+		/// The call filter that will be used when pallet is inactive
+		type InactiveFilter: Contains<<Self as frame_system::Config>::Call>;
 	}
 
 	#[pallet::hooks]
 	impl<T: Config> Hooks<BlockNumberFor<T>> for Pallet<T> {}
 
-	/// Pallet genesis configuration type declaration.
-	///
-	/// It allows to build genesis storage.
-	#[pallet::genesis_config]
-	pub struct GenesisConfig {}
-
-	#[cfg(feature = "std")]
-	impl Default for GenesisConfig {
-		fn default() -> Self {
-			Self {}
-		}
+	#[pallet::type_value]
+	pub fn OnCompletedEmpty() -> Status {
+		Status::Inactive
 	}
```

Comment:
Ok so this is actually a "default" value for this type. You could implement the `Default` trait. Leave it up to you.
But we do need to update this:

```suggestion
	pub fn DefaultMigrationStatus() -> Status {
		MigrationStatus::Inactive
	}
```

Reply:
No, I want it explicitly to be OnEmpty. As there is no default, just a storage default in case storage is empty.

Reply:
Yes that's a good point 👍